server/service/src/invoice_line/outbound_shipment_unallocated_line/allocate/generate.rs

```diff
@@ -192,10 +192,10 @@ fn packs_to_allocate_from_stock_line(remaining_to_allocated: i32, line: &StockLi
     let fractional_number_of_packs = remaining_to_allocated as f64 / line_row.pack_size as f64;
 
     if fraction_is_integer(fractional_number_of_packs) {
-        return fractional_number_of_packs as i32;
+        return fractional_number_of_packs;
     }
```

Comment:
not required now? 
we did talk about the intention here and I thought we were planning to retain the allocation of whole packs, but express the result as f64. not sure.. maybe a TODO to review?

Side nit: `remaining_to_allocated` would be better as `remaining_to_allocate` or `remaining_to_be_allocated`

Reply:
✌️ [9769a86](https://github.com/openmsupply/open-msupply/pull/642/commits/9769a86127fcabd8514ae59bc1aeaeb1792d1765)

Reply:
As per Marks comment, I think `floor()` + 1 should be retained ? And the test that was deleted should still pass, I think you can link that issue you created to the comment too